src/components/PageHeader.tsx

```diff
@@ -15,17 +15,17 @@ const StyledHeader = styled.div`
   }
```

Comment:
Må ha noe settings på overflow-wrap/word-break/etc for at lange headings skal virke. Feks bør "Personvernerklæring" på mobil brytes med en "-", ellers burde det brytes på ord

Reply:
får ikke inn hyphens automatisk :( 
![bilde](https://user-images.githubusercontent.com/6313468/106724769-ea344e00-6608-11eb-90b6-e69b3eb84241.png)

Reply:
Fikser det nok i min branch på en eller annen måte!